hack/make-rules/test-cmd.sh

```diff
@@ -720,6 +720,24 @@ runTests() {
   # Pre-condition: no POD exists
   create_and_use_new_namespace
   kube::test::get_object_assert pods "{{range.items}}{{$id_field}}:{{end}}" ''
+  ## kubectl create --edit can update the image field of a POD. tmp-editor.sh is a fake editor
+  echo -e "#!/bin/bash\n$SED -i \"s/gcr.io\/google_containers\/serve_hostname/nginx/g\" \$1" > /tmp/tmp-editor.sh
```

Comment:
Use `mktemp` instead of creating a file that might conflict with something else.